docs/api/api-spec.md

```diff
@@ -132,6 +132,46 @@ On the financial desktop, applications often want to broadcast context to any nu
 ## Resolvers
 Intents functionality is dependent on resolver functionality to map the intent to a specific App.  This will often require end-user input.  Resolution can either be performed by the Desktop Agent (raising UI to pick the desired App for the intent) or by the app launching the intent - in which case the calling App will handle the resolution itself (using the findIntents API below) and then invoke an explicit Intent object.
 
+## Context channels
+
```

Comment:
In general I think we should keep the spec relatively high level - it should explain intention in clear terms, without too many code samples beyond making that intention clear, and we should try to keep technical terms like `dev/null` out of it.

The right places for usage samples is in the `examples` folder and code comments I think. (I realise that we don't currently have other API usage samples.)

Reply:
dev/null has been sent to dev/null